releaseVersionUpdate.py

```diff
@@ -0,0 +1,36 @@
+import json
+import requests
+import sys
+from xml.dom import minidom
+
+sasUrl = sys.argv[1]
+releaseVersion = sys.argv[2].split(' ')[2]
+print('Release Version: ' + releaseVersion)
+if(len(releaseVersion)==0):
+    print('Incorrect Release Version')
+    sys.exit(1)
+
+containerUrl = sasUrl.split('?')[0]
+sasToken = sasUrl.split('?')[1]
+
+# list latest version file in the container
+listUrl = sasUrl + '&restype=container&comp=list&prefix=latest/'
```

Comment:
why not use the Python SDK for some of this code?

Reply:
then we need to clone/build that as well on build machine?